data/Witkowski_1991/metadata.yml

```diff
@@ -0,0 +1,257 @@
+    volume: '39'
+    number: '4'
+    pages: '385'
+    doi: 10.1071/bt9910385
+contributors:
+  data_collectors:
+  - last_name: Witkowski
+    given_name: E. T.F.
+    ORCID: .na
+    affiliation: University of the Witwatersrand, South Africa
+    notes: no contact
+  austraits_curators: Elizabeth Wenk
+dataset:
+  data_is_long_format: no
+  custom_R_code:   '
+    data %>%
+      mutate(
+        location = "Hopetoun"
+      )
+  '
+  collection_date: 1998-04
+  taxon_name: Species
+  location_name: location
+  description: study of seed bank dynamics of three co-occurring, non-sprouting Banksia
+    species in the southern sandplains of Western Australia, to compare to seed bank
+    dynamics of non-sprouting Banksia species in the northern sandplains.
+  basis_of_record: field
+  life_stage: adult
+  sampling_strategy: Study Sites ^p The study site was located on a dune system of
+    deep infertile siliceous sands, 26 km E. of the south coastal town of Hopetoun,
+    W.A. and 1-2km inland from the coast (33" 57'S.,120° 23'E., 20m a.s.1.). The site
+    was burnt about 10 years ago by a patchy (probably spring) burn which resulted
+    in intermingled remnants of the parent population which were 21 years old at the
+    time of the study (stem increment counts, Lamont 1985). A second site, 4 k m E.
+    of the first, consisted of a stand of only 21-year-old plants. The vegetation
+    was Banksia scrub-heath about 3 m in height, consisting of evergreen sclerophyllous
+    trees, shrubs and herbs. The length of the growing season is 7-8 months compared
+    with 4-5 months for the northern sandplains (Bartlett 1975). Annual rainfall is
+    528 mm (range 376-687 mm), and mean maximum and minimum temperatures are 25.7OC
+    and 7.7OC respectively. Soils are at least 10m deep, infertile, mildly acidic
+    (pH 5.9 5 0.2), siliceous sands
+  original_file: none, transcribed from manuscript
+  notes: none
+locations:
+  Hopetoun:
+    latitude (deg): -33.95
+    longitude (deg): 120.383
+    description: dune system of deep infertile siliceous sands
+    locality: Hopetoun, Western Australia
+    fire history: The site was burnt about 10 years ago by a patchy (probably spring)
+      burn which resulted in intermingled remnants of the parent population which
+      were 21 years old at the time of the study (stem increment counts, Lamont 1985).
+      A second site, 4 k m E. of the first, consisted of a stand of only 21-year-old
+      plants.
+    vegetation type: Banksia scrub-heath about 3 m in height, consisting of evergreen
+      sclerophyllous trees, shrubs and herbs
+    soil type: Soils are at least 10m deep, infertile, mildly acidic (pH 5.9 5 0.2),
+      siliceous sands.
+contexts: .na
+traits:
+- var_in: reproductive_maturity
+  unit_in: a
+  trait_name: reproductive_maturity
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: .na
```

Comment:
Should `replicates` also be `replicates_seed_fruit`, like in `fire_time_from_fire_to_flowering`?

Reply:
done